src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
+
+import { Movie } from '../../types/Movie';
+
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+import { MovieCard } from '../MovieCard';
+
+export const MoviesList: React.FC<{ moviesFromServer: Movie[] }> = ({
+  moviesFromServer,
+}) => {
+  return (
+    <>
+      {moviesFromServer.map((movie) => (
+        <MovieCard movie={movie} />
```

Comment:
don't forget about key by mapping
```suggestion
        <MovieCard key={movie.imdbId} movie={movie} />
```